client/client_test.go

```diff
@@ -81,7 +83,7 @@ func newNotifyingSender(wrapped client.KVSender) *notifyingSender {
 // an HTTP sender to the server at addr.
 // It contains a waitgroup to allow waiting.
 func createTestNotifyClient(addr string) *client.KV {
-	sender, err := client.NewHTTPSender(addr, testutils.NewTestBaseContext())
+	sender, err := http.NewHTTPSender(addr, testutils.NewTestBaseContext())
```

Comment:
What do you think of unexporting the HTTPSender and RPCSender types themselves in this PR? Does not look like breaks anything.